bpf/lib/nodeport.h

```diff
@@ -1542,6 +1542,7 @@ int tail_nodeport_nat_ipv4(struct __ctx_buff *ctx)
 		}
 	}
 #endif
+	/* Handle reverse SNAT for reply packets from remote backends. */
```

Comment:
But this comment is a bit misleading given it's handling both, forward and reverse NAT.

Reply:
If `NAT_DIR_INGRESS` - then rev-SNAT, if `NAT_DIR_EGRESS` - then SNAT.

Reply:
Fixed.